src/actions/index.js

```diff
@@ -273,3 +273,48 @@ export function toggleHoveredFocusMode(hoveredFocusMode) {
     hoveredFocusMode,
   };
 }
+
+export const TOGGLE_BOOKMARK = 'TOGGLE_BOOKMARK';
+
+/**
+ * Toggle bookmark of Run
+ * @param {boolean} bookmark
+ * @param {string} RunId
```

Comment:
[Nit] can we follow the camelCase for parameters

Reply:
Done.